json-schema/json-schema/src/main/scala/endpoints4s/Validated.scala

```diff
@@ -110,4 +110,18 @@ object Validated {
       case Failure(error) => Invalid(error.getMessage)
     }
 
+  /** Turns a `Seq[Validated[A]]`into a `Validated[Seq[A]]`.
+    * If all values in the sequence are `Valid`, the result will be `Valid`
+    * If there is at least one `Invalid`, the result will be an `Invalid` containing the error messages of all `Invalid`s
+    * If the `Seq` is empty, will return a `Valid[Seq.empty]`
+    */
+  def fromSeq[A](seq: Seq[Validated[A]]): Validated[Seq[A]] = {
```

Comment:
I think we should have a signature similar to the one of `Future.sequence`, to support any collection, not just `Seq`:

https://github.com/scala/scala/blob/99de4622e8e81e0adc4d4ea60f81bc42d40f84b9/src/library/scala/concurrent/Future.scala#L709

That would also be more efficient (constructing a collection with a `Builder` is the best approach to be collection agnostic).